packages/react-data-grid/src/common/types.ts

```diff
@@ -117,17 +117,17 @@ export interface Editor {
   readonly disableContainerStyles?: boolean;
 }
 
-export interface FormatterProps {
+export interface FormatterProps<V, T = unknown> {
   rowIdx: number;
-  value: unknown;
+  value: V;
   column: CalculatedColumn;
   row: RowData;
   isScrolling: boolean;
-  dependentValues?: unknown;
+  dependentValues?: T;
 }
 
 export interface EditorProps<V = unknown, C = unknown> {
-  column: CalculatedColumn<C>;
+  column: CalculatedColumn<V, C>;
   value: V;
   rowMetaData: unknown;
```

Comment:
`rowMetaData` should not be a required field.
https://github.com/adazzle/react-data-grid/blob/next/packages/react-data-grid/src/common/editors/EditorContainer.tsx#L182

Reply:
Also it should be `C` or `DV` (for clarity?)